src/app/services/openshift.service.ts

```diff
@@ -0,0 +1,32 @@
+import { init } from '@aerogear/app';
+import { Injectable } from '@angular/core';
+
+const config = require('../../mobile-services.js');
+
+@Injectable({
+  providedIn: 'root'
+})
+/**
+ * Service that initializes OpenShift specific SDK's
+ */
+export class OpenShiftService {
+
+  constructor() {
+    init(config);
+  }
+
+  getConfig() {
+    return config;
+  }
+
+  hasSyncConfig() {
+    return config.services.filter((service) =>
```

Comment:
It's a nitpicky thing but considering this is an example app it would be good to keep the code simple.

Would be nice to use `array.find` here

```js
return config.services.find((service) => service.name === 'sync')
```